dashboard/scripts/turn_off_stage_extras_for_non_csf.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+# Update all non-CSF sections that have Lesson Extras enabled to turn off Lesson Extras.
+require_relative('../config/environment')
+
+puts 'Starting to batch update all non-CSF sections with Lesson Extras enabled.'
+puts "Total sections: #{Section.count}"
+
+num_sections_scanned = 0
+num_sections_updated = 0
+
+batch_size = 10000
+
+Section.where.not(stage_extras: false, script_id: Script.csf_script_ids).in_batches(of: batch_size) do |sections|
+  values = sections.pluck(:id, :stage_extras, :user_id)
```

Comment:
There's no need to pluck the existing `:stage_extras` value since you're just clobbering it, and I don't see the need to include `:user_id` in this operation either since it doesn't get updated.  I think this will work?

```rb
values = sections.pluck(:id).map do |id|
  [id, false]
end
result = Section.import([:id, :stage_extras], values, validate: false, on_duplicate_key_update: [:stage_extras])
```

Reply:
Good call on not plucking `:stage_extras`, I will update!

For `:user_id`, I added that to resolve the following error, since that column has no default value:

```
[40] pry(main)> Section.import([:id, :stage_extras], [[70, false]], validate: false, on_duplicate_key_update: [:stage_extras])
ActiveRecord::StatementInvalid: Mysql2::Error: Field 'user_id' doesn't have a default value: INSERT INTO `sections` (`id`,`stage_extras`,`created_at`,`updated_at`,`deleted_at`) VALUES (70,0,'2018-04-11 18:12:57','2018-04-11 18:12:57',NULL) ON DUPLICATE KEY UPDATE `sections`.`stage_extras`=VALUES(`stage_extras`),`sections`.`updated_at`=VALUES(`updated_at`)
```

Reply:
Interesting.  So it has to be provided since it's an 'upsert' even though we expect to only update existing rows?

Reply:
@wjordan pointed out the query should only modify the row(s) specified in `on_duplicate_key_update`, so in theory we could set any arbitrary value for `:user_id` in the `Section.import` statement.  Using the real value feels a little safer to me.